app/code/Magento/Customer/Controller/Adminhtml/Index/InlineEdit.php

```diff
@@ -179,7 +179,7 @@ protected function getData(array $data, $isCustomerData = null)
         $addressKeys = preg_grep(
             '/^(' . AttributeRepository::BILLING_ADDRESS_PREFIX . '\w+)/',
             array_keys($data),
-            $isCustomerData
+            $isCustomerData ?? 0
```

Comment:
Hello @karyna-tsymbal-atwix 
The $isCustomerData is mixed param, however `preg_grep` function requires integer there.
Perhaps, we need to cast it.
```suggestion
            (int) $isCustomerData ?: 0
```

or

```suggestion
            is_int($isCustomerData) ? $isCustomerData : 0
```
What do you think about it?
cc @xmav

Reply:
`(int) $isCustomerData` should cover current argument value variations:[null, true] and other possible variations(if any)

Reply:
I replaced it with (int) cast